Improvements for variable names

danielgindi opened the pull request.

aviaryan commented on BigEval.js:
```diff
@@ -298,12 +295,34 @@ BigEval.prototype._tokenizeExpression = function (expression) {
+
+			while (i < len) {
+				c = expression[i];
+                isVarChars = this.varNameChars[c];
```
I saw that our project was a mess (as seen in this line) when it came to indentation so I have fixed on following [happiness](https://github.com/JedWatson/happiness) guidelines in the project. 
Please rebase with master and then the new checks should apply on this PR too.

danielgindi replied:
Done :-)

aviaryan commented:
@danielgindi Thanks for the quick work. Unfortunately, a test fails, please check.

danielgindi commented:
Seems OK now.
There was one test that failed even with master, due to native `eval` not handline the `**` operator then the l-value is a negative number.
It gave "Invalid unary operator on the left hand side of exponentiation (**) operator".

I've fixed that, and fixed another issue with the exponentiation operator along the way (When there's multiple **, like `2**7**-3`), where the order should be reversed.

danielgindi commented:
Btw, you may want to add a link to the port https://github.com/danielgindi/Eval.net
